examples/glasbey-colormap.py

```diff
@@ -0,0 +1,81 @@
+"""
+Glasbey colormap
+================
+
+A long-requested feature [1]_ for napari was to display labels/segmentation
+layers using a well-known discrete colormap such as Glasbey [2]_.
+
+In this example, we demonstrate displaying segmentations using custom colormaps
+with the help of the glasbey Python library [3]_. We display a segmentation
+using the napari built-in labels colormap, the original Glasbey colormap, a
+more modern version produced by limiting the lightness and chroma and
+optimizing for colorblind-safety, and finally with the matplotlib tab10
+colormap.
+
+.. [1] https://github.com/napari/napari/issues/454
+.. [2] Colour displays for categorical images. Chris Glasbey, Gerie van der
+       Heijden, Vivian F. K. Toh, and Alision Gray. (2007)
+       :DOI:`10.1002/col.20327`
```

Comment:
I think if we remove the `:DOI:` role that this will pass. Not sure if this is a role that we support.

Reply:
Oh! We've had this in skimage for so long that I thought it was built into sphinx! 😂 

This is what we would need to use it:

https://github.com/scikit-image/scikit-image/pull/3367/files#diff-f31881d2140a5aa15bb25932e98afa77b57930e1c589ca9785c82c1f92a718ae

Since it's not even a properly packaged extension, I am going to go ahead and remove it. Thanks for spotting it @willingc!

Reply:
```suggestion
       DOI:10.1002/col.20327
```